java/test/jmri/jmrit/display/layoutEditor/LayoutSlipTest.java

```diff
@@ -30,15 +459,23 @@ public void setUp() throws Exception {
         // dispose of the single PanelMenu instance
         jmri.jmrit.display.PanelMenu.dispose();
         // reset the instance manager.
-        JUnitUtil.resetInstanceManager();
+        jmri.util.JUnitUtil.resetInstanceManager();
+        ///DOMConfigurator.configure("myapp-log4j.xml");
+        apps.tests.Log4JFixture.initLogging();
```

Comment:
`setUp()` should initialize Log4J (really, the entire logging infrastructure) _before_ it does _anything_ else.  Almost anything in the JMRI code can do logging operations.  Once something does, that forces a Log4J default initialization, and it's no longer using the `tests.lcf` logging definition, which in turn will lead to various confusing things.